### ISSUE TITLE
Broken with android gradle plugin 1.1.2 (Error:(51, 0) No such property: bootClasspath for class: com.android.build.gradle.AppPlugin)

### ISSUE THREAD

**Issue by thoutbeckers** [novoda/gradle-android-test-plugin#41]
It seems c8e006e4ace3c0555c77c4dff7b38d541a3ce12e was made to fix 1.1.0 compatibility by getting this property elsewhere but then 1.1.1 was release with this changelog note:

> Calling android.getBootClasspath() is now possible again.

**Comment by thoutbeckers** [comment]
A working solution seems to be
```groovy
 def androidRuntime = projectUnderTest.files(projectUnderTest.android.getBootClasspath())
```
I'll see if I can PR this today.

**Comment by nenick** [comment]
Do you have any issue when you use `gradle plugin 1.1.2` and `novoda plugin 0.10.3`? I don't see this error message for my sample project https://github.com/nenick/android-gradle-template

**Comment by thoutbeckers** [comment]
See PR #42 

Also it seems this is the recommended way since it's a public API: https://groups.google.com/d/msg/adt-dev/vhjDsuQmG7w/IuU_iJan3cMJ

**Comment by nenick** [comment]
Using public API instead of non public is good reason to change this.

**Comment by nenick** [comment]
closed in favour for #43

**Comment by dschuermann** [comment]
As this is now resolved, could you please release a new version?

**Comment by blundell** [comment]
yes, hopefully within 1 - 5 days

**Comment by dschuermann** [comment]
Awesome thanks!

**Comment by blundell** [comment]
Released `0.10.4` https://github.com/novoda/gradle-android-test-plugin/releases/tag/0.10.4
